### ISSUE TITLE
Refine itinerary Selection - Center pane header flight selection

### ISSUE THREAD

**Issue by GrosR** [clowdfish/ttg-frontend#108]
As already said in issue #105 , the center screen in the refine itinerary section should always tell the user what he is doing at the moment. In the flight upgrade screen, the header in the center pane should show the same appointment details (appointment name, description, date) as in the other screens. 

The description of the action to be taken should be: 

Your chosen route offers upgrade options. Select your preferred option by "Click + Next-Button" or "Double click"

![image](https://cloud.githubusercontent.com/assets/11087531/17519467/56982572-5e4c-11e6-9b51-ccb5b7893314.png)

**Comment by davidtran** [comment]
where should I place that text, above the map ?

**Comment by GrosR** [comment]
yes, where you see the date/time and "Stuttgart, Germany to Hannover, Germany" Header right now, we should write the appointment data, in this Sample: 
16.08.16, 9 AM - 6 PM
CRM Besprechung, Marquardt Service GmbH
Below this Heading, we should add 
Your chosen route "Flight from Stuttgart, Germany to Hannover, Germany " offers upgrade options. Select your preferred option by "Click + Next-Button" or "Double click"

The Content "CRM Besprechung, Marquardt Service GmbH" is of course a variable filled from the "Meeting Name" and "Destination Name" of currently active appointment. 

As the destination contains meeting description + location address , we should shorten this to "meeting description + location city"

The Content "Flight from Stuttgart, Germany to Hannover, Germany " is of course a variable filled with the data from the chosen route.
